scripts/azure/SharePoint/sharepoint_scanner.py

```
@@ -0,0 +1,2232 @@
+        self.load_libraries_button.setEnabled(True)
+        self.load_folders_button.setEnabled(True)
+        self.site_url_input.setEnabled(True)
+        self.output_folder_input.setEnabled(True)
+        self.skip_count_checkbox.setEnabled(True)
+        self.scan_worker = None
+        self.scan_start_time = None
+
+    def _open_output(self):
+        folder = self.output_folder_input.text().strip()
+        if folder and Path(folder).exists():
+            os.startfile(folder)
```

Ruff (0.15.7)

[error] Starting a process without a shell

(S606)